### ISSUE TITLE
10.1.1- Message toasts sending to wrong players

### ISSUE THREAD

**Issue by NerdyBirder** [kangarko/ChatControl#172]
Toasts from dm messages are being sent to random players in addition to the player receiving the dm message. 

Can not find any errors or tip messages from the console but i didn't have much time to really check.

I have disabled toasts on my server so that i don't have to worry about messages being shown to the wrong players for now but could enable it again if its needed to test.

**Comment by kangarko** [comment]
1. Do you have MySQL on?
2. BungeeCord on?
3. Does it happen over bungee only?

**Comment by kangarko** [comment]
Try next release, let me know below if that wont help.

**Comment by NerdyBirder** [comment]
Yes it was over bungee, yes we use mysql.
Ok, will try the new release and test. I will let you know if it is still an issue.

**Comment by NerdyBirder** [comment]
Ok, so i did a bit more testing of this issue tonight after putting in 10.1.2.

I only enabled toasts in 2 of our bungee instances and had someone message back and forth to me. One of my opped staff told me they were seeing some of the toasts in a bungee instance that still had toasts disabled in the settings.yml while the person helping me test and I were not getting any toasts until we went into the same bungee instance and sent pms there. So the toasts are getting lost in bungee somewhere and being sent to the wrong person sometimes or not at all.

I disabled toasts again which is probably where i am going to leave it for my server from now on.
